nixos/modules/services/misc/paperless.nix

```
@@ -252,9 +252,59 @@ in
         '';
       };
     };
+
+    exporter = {
+
+      enable = lib.mkEnableOption "regular automatic document exports.";
+
+      directory = lib.mkOption {
+        type = lib.types.str;
+        default = cfg.dataDir + "/exports";
+        description = lib.mdDoc "Directory to store exports.";
```

Nit: mdDoc is deprecated; it's the default now.

```suggestion
        description = "Directory to store exports.";
```